README.md

```diff
@@ -9,6 +9,13 @@
 5. Create a PR
 6. Profit
 
+### Server side hot reloading
+
+There is a Nodemon config for server side hot loading.
+
+1. Install the Nodemon CLI tool: `npm install -g nodemon`
```

Comment:
Since we already have nodemon installed in our packages, we can rely on it by invoking nodemon from the package.json as a command (preferably the `dev` command should use nodemon). This way we don't have to install global packages (which is a nice thing)

Reply:
Fixed